addons/pos_loyalty/tests/test_frontend.py

```diff
@@ -342,9 +342,9 @@ def test_loyalty_free_product_loyalty_program(self):
 
         (self.promo_programs | self.coupon_program).write({'active': False})
 
+        partner_ccc = self.env['res.partner'].create({'name': 'Test Partner CCC'})
```

Comment:
don't move this line. we should keep the original order. besides this change has no effect.